test.cpp

```
@@ -0,0 +1,48 @@
+    int result = uninitialized + 7;
+
+    int x = divide(100, 0);
+
+    int* p = NULL;
+    nullDeref(p);
+
+    Shape* s = new Circle();
+    s->draw();
+
+    const char* fmt = "%s";
+    printf(fmt);
```

Cppcheck (2.20.0)

[error] printf format string requires 1 parameter but only 0 are given.

(wrongPrintfScanfArgNum)